package-lock.json

```diff
@@ -1,18 +1,18 @@
 {
-  "name": "cdk-pgstac",
```

Comment:
this still had `cdk-pgstac` instead of `eoapi-cdk` in it ? 😵‍💫

Reply:
oh it's interesting, I pushed my own package-lock.json while it seems that when doing a release only the version number was changed in the package-lock.json, not the whole file was update 🤷‍♂️